custom_components/acwd/acwd_api.py

```
@@ -70,17 +76,17 @@ def login(self):
         if not csrf_token:
-            logger.error("No CSRF token found!")
+            _LOGGER.error("No CSRF token found!")
             return False
 
         # Store CSRF token for future requests
         self.csrf_token = csrf_token
-        logger.debug("CSRF token obtained")
+        _LOGGER.debug("CSRF token obtained")
 
         # Step 3: Call updateState endpoint
         logger.info("Calling updateState endpoint...")
```

Ruff (0.14.10)

Undefined name `logger`

(F821)